google/ggrc-core
# GGRC-4977 Remove 'Manager' column for program

# Issue description

Extra 'Manager' column is displayed in Global Search/Unified mapper/Tree view

# Steps to test the changes

1. Open My work page
2. Open Global Search
3. Select 'Program' object type
4. Click Search button
5. Set visible fields for Program > Select checkboxes for Manager and Program managers columns

**Actual Result:** Extra Manager column duplicates data of Program managers column in Global Search/Unified mapper
**Expected Result:** Remove 'Manager' column for Program in Set visible fields popup in:

- Global Search
- Unified mapper
- Tree view
- Export/import template

# Solution description

Update Program model config.

# Sanity checklist




# PR Review checklist
